src/containers/ConnectorManagement.jsx

```diff
@@ -190,16 +192,16 @@ export default class ConnectorManagement extends Component {
   async updateAccount (connector, idx, values) {
     const { t } = this.context
 
-    const modifiedAccount = {
+    const modifiedAccount = Object.assign({}, connector.accounts[idx], {
       auth: {
         login: values.login,
         password: values.password
       }
-    }
+    })
     const folder = await cozy.client.files.statById(connector.accounts[idx].folderId, false, {limit: 1})
     const folderPath = folder.attributes.path
 
-    return this.store.connectAccount(connector, modifiedAccount, folderPath)
+    return run(cozy.client, connector, modifiedAccount)
```

Comment:
Same thing here, a call to run should be abstracted into the store, we don't have to do any import from libs in our components. Just call the store and it is then dealing with encapsulated libraries.